### ISSUE TITLE
Automate WordPress (blog) deployment

### ISSUE THREAD

**Issue by renoirb** [webplatform/ops#32]
WordPress should be only a few commands to deploy.
## Tasks
- [ ] Automation setup, once figured out in webplatform/ops#80
## Expected workflow

```
git clone https://github.com/webplatform/blog.git
cd blog
make
```
## Related
- webplatform/ops#80
- webplatform/ops#22

**Comment by renoirb** [comment]
Duplicate of #22

**Comment by renoirb** [comment]
Depends on #22 but is only about the automation.

**Comment by renoirb** [comment]
This should be done now.
